### ISSUE TITLE
Queue is not working

### ISSUE THREAD

**Issue by RomLecat** [devopsarr/sonarr-py#6]
Hello,

The Sonarr queue API is not working properly.

```python
    config = sonarr.Configuration(host=url)
    config.api_key['apikey'] = api_key
    with sonarr.ApiClient(config) as api_client:
        queue = sonarr.QueueApi(api_client).get_queue(include_unknown_series_items=True, include_series=True, include_episode=True)
        print(queue)
```

I get this error:

```
Traceback (most recent call last):
  File "/opt/zabbix-tools/./arr_agent.py", line 65, in <module>
    main()
  File "/opt/zabbix-tools/./arr_agent.py", line 41, in main
    process_sonarr(url, api_key, host['host'])
  File "/opt/zabbix-tools/./arr_agent.py", line 54, in process_sonarr
    queue = sonarr.QueueApi(api_client).get_queue(include_unknown_series_items=True, include_series=True, include_episode=True)
  File "pydantic/decorator.py", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_function
  File "pydantic/decorator.py", line 134, in pydantic.decorator.ValidatedFunction.call
  File "pydantic/decorator.py", line 206, in pydantic.decorator.ValidatedFunction.execute
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api/queue_api.py", line 384, in get_queue
    return self.get_queue_with_http_info(include_unknown_series_items, include_series, include_episode, **kwargs)  # noqa: E501
  File "pydantic/decorator.py", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_function
  File "pydantic/decorator.py", line 134, in pydantic.decorator.ValidatedFunction.call
  File "pydantic/decorator.py", line 206, in pydantic.decorator.ValidatedFunction.execute
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api/queue_api.py", line 490, in get_queue_with_http_info
    return self.api_client.call_api(
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 407, in call_api
    return self.__call_api(resource_path, method,
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 206, in __call_api
    url_query = self.parameters_to_url_query(query_params,
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 546, in parameters_to_url_query
    return "&".join(["=".join(item) for item in new_params])
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 546, in <listcomp>
    return "&".join(["=".join(item) for item in new_params])
TypeError: sequence item 1: expected str instance, bool found
```

Radarr has the exact same issue.

Thanks

**Comment by Fuochi** [comment]
Hi @RomLecat,
it seems I'm not managing non-string query parameters properly, I'll address this after #5 
Thanks

**Comment by RomLecat** [comment]
Hi,

I did not close this one, I just had another error with the queue (it currently has 21 pending items, with 12 stalled ones due to missing peers in torrents) :

```
Traceback (most recent call last):
  File "/opt/zabbix-tools/arr_agent.py", line 91, in <module>
    main()
  File "/opt/zabbix-tools/arr_agent.py", line 47, in main
    process_arr(host, sonarr)
  File "/opt/zabbix-tools/arr_agent.py", line 75, in process_arr
    queue = api_soft.QueueApi(api_client).get_queue(include_unknown_series_items=True, include_series=>
  File "pydantic/decorator.py", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_fun>
  File "pydantic/decorator.py", line 134, in pydantic.decorator.ValidatedFunction.call
  File "pydantic/decorator.py", line 206, in pydantic.decorator.ValidatedFunction.execute
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api/queue_api.py", line 384, in get_queue
    return self.get_queue_with_http_info(include_unknown_series_items, include_series, include_episode>
  File "pydantic/decorator.py", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_fun>
  File "pydantic/decorator.py", line 134, in pydantic.decorator.ValidatedFunction.call
  File "pydantic/decorator.py", line 206, in pydantic.decorator.ValidatedFunction.execute
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api/queue_api.py", line 490, in get_queue_with_h>
    return self.api_client.call_api(
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 409, in call_api
    return self.__call_api(resource_path, method,
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 250, in __call_api
    return_data = self.deserialize(response_data, response_type)
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 320, in deserialize
    return self.__deserialize(data, response_type)
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 359, in __deserialize
    return self.__deserialize_model(data, klass)
  File "/usr/local/lib/python3.9/dist-packages/sonarr/api_client.py", line 751, in __deserialize_model
    return klass.from_dict(data)
  File "/usr/local/lib/python3.9/dist-packages/sonarr/models/queue_resource_paging_resource.py", line >
    "records": [QueueResource.from_dict(_item) for _item in obj.get("records")] if obj.get("records") >
  File "/usr/local/lib/python3.9/dist-packages/sonarr/models/queue_resource_paging_resource.py", line >
    "records": [QueueResource.from_dict(_item) for _item in obj.get("records")] if obj.get("records") >
  File "/usr/local/lib/python3.9/dist-packages/sonarr/models/queue_resource.py", line 189, in from_dict
    _obj = QueueResource.parse_obj({
  File "pydantic/main.py", line 526, in pydantic.main.BaseModel.parse_obj
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for QueueResource
timeleft
  expected string or bytes-like object (type=type_error)
```

**Comment by RomLecat** [comment]
Just did some tests, so far so good, thanks.
I'll close this one since it looks it works fine now, I'll open a new issue if I encounter something else :)
